### DISCUSSION TITLE
[Awards][WAS] map view

### DISCUSSION THREAD

**Discussion by hairlesshobo** [magicbug/Cloudlog#733] [General]
I was wondering if it would be possible for you to add a map view to the WAS awards page. I envision the map being sort of like the grid map, except that the state would be outlined and shaded either red/yellow/green depending on whether the status is not worked/worked/confirmed. 

Thanks in advance for your time! 

73s 
Steve

**Comment by AndreasK79** [comment]
Peter and I discussed this, and it is something we talked about adding. I guess it can go on the wish-list.

**Comment by magicbug** [comment]
Yep still on my list of things to add either using Leafetjs or Highcharts (Need to check licencing on this)

**Comment by magicbug** [comment]
Possible library found today is https://datamaps.github.io looks quite useful for other ideas of Cloudlog too

**Comment by hairlesshobo** [comment]
That library looks awesome! I will have to keep that in mind for some of my future projects too. I could see you making some awesome map views in Cloudlog with that.

Side note, I meant to include this in the original post. I love Cloudlog. You guys are doing a fantastic job on it, please keep it up!!

**Comment by AndreasK79** [comment]
@magicbug I think you found the map to use for this :)

**Comment by magicbug** [comment]
As per this https://github.com/magicbug/Cloudlog/discussions/1008 and the commits to the codebase this is now complete
